projects/packages/wpcom-posts/license.txt

```diff
@@ -0,0 +1,14 @@
+This program is free software; you can redistribute it and/or
```

Comment:
We haven't typically added licenses to our packages until now, but you make a good point. I've asked in p2y3YZ-4v3-p2